x-pack/plugins/cloud_security_posture/public/pages/rules/rules_table.tsx

```diff
@@ -190,7 +193,7 @@ const getColumns = ({
         }}
       />
     ),
-    width: '3%',
+    width: '40px',
```

Comment:
40px seems too much for a 16x16 checkbox, can you add a screen shot?

Reply:
I have updated the screenshot 👍